assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -297,6 +301,9 @@ che.openshift.workspaces.pvc.quantity=10Gi
 che.openshift.workspace.cpu.limit=1
 # Override memory limit used for openshift workspaces. String, e.g. 1300Mi
 che.openshift.workspace.memory.override=NULL
+# Create secure route against HTTPS 
```

Comment:
@l0rd but than there will be dependency between evaluation strategy and openshift. Maybe having two properties make more sense for the sake of decoupling ?